database/sql/V016__update_order_indexes.sql

```diff
@@ -0,0 +1,6 @@
+-- The following index already allows indexing by owner.
+DROP INDEX order_owner;
```

Comment:
Should we also drop `order_creation_timestamp` index as well?

Reply:
That is going to be used in the future and is different from the index on (owner, creation_timestamp) because the latter doesn't allow you to order orders just by creation_timestamp.